pkg/sql/vars.go

```diff
@@ -3801,6 +3801,23 @@ var varGen = map[string]sessionVar{
 			return formatBoolAsPostgresSetting(kvcoord.BufferedWritesEnabled.Get(sv))
 		},
 	},
+
+	// CockroachDB extension.
+	`opt_fk_uniq_check_lower_bound_enabled`: {
+		Get: func(evalCtx *extendedEvalContext, _ *kv.Txn) (string, error) {
+			return formatBoolAsPostgresSetting(evalCtx.SessionData().OptFKUniqCheckLowerBoundEnabled), nil
+		},
+		GetStringVal: makePostgresBoolGetStringValFn("opt_fk_uniq_check_lower_bound_enabled"),
+		Set: func(ctx context.Context, m sessionDataMutator, s string) error {
+			b, err := paramparse.ParseBoolVar(`opt_fk_uniq_check_lower_bound_enabled`, s)
+			if err != nil {
+				return err
+			}
+			m.SetOptFKUniqCheckLowerBoundEnabled(b)
+			return nil
+		},
+		GlobalDefault: globalTrue,
```

Comment:
Backports can be simpler if you set this to false in the first commit, and set it to true in another commit. Then you can backport just the first commit cleanly without having to update pg_catalog, show_source, and information_schema tests.

Reply:
I went ahead and did this.

Reply:
Of course, the backports weren't clean because I didn't do the smart thing you did with the session setting orderings. Oh well... They are here if we want them:

https://github.com/cockroachdb/cockroach/pull/141374
https://github.com/cockroachdb/cockroach/pull/141375
https://github.com/cockroachdb/cockroach/pull/141376
https://github.com/cockroachdb/cockroach/pull/141378